src/utils/test-utils.ts

```diff
@@ -0,0 +1,21 @@
+import * as request from 'supertest';
+import { INestApplication, ModuleMetadata } from '@nestjs/common/interfaces';
+import { Test } from '@nestjs/testing';
+import { MongooseModule } from '@nestjs/mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+
+export async function testModuleWithInMemoryDb(moduleMetadata: ModuleMetadata) {
+  const mongoServer = new MongoMemoryServer();
+  const uri = await mongoServer.getConnectionString();
```

Comment:
Tools in my editor complain it's deprecated. Try to use `grtUri(...` instead.

Reply:
Fixed.  Secon deprecation warning should be fixed with the next mongoose release.